Resources/Prototypes/_StarLight/Entities/Mobs/Species/thaven.yml

```
@@ -113,6 +113,11 @@
           32:
             sprite: _Starlight/Mobs/Species/Thaven/displacement.rsi
             state: neck
+      misc:
+        sizeMaps:
+          32:
+            sprite: _Starlight/Mobs/Species/Cyclorite/displacement.rsi
```

it should be since it is thaven displacements.
```suggestion
            sprite: _Starlight/Mobs/Species/Thaven/displacement.rsi
```